src/prog_models/prognostics_model.py

```diff
@@ -598,7 +601,61 @@ def threshold_met(self, x) -> dict:
         return {key: event_state <= 0 \
             for (key, event_state) in self.event_state(x).items()} 
 
-    def simulate_to(self, time : float, future_loading_eqn : Callable, first_output = None, **kwargs) -> namedtuple:
+    @property
+    def is_direct_model(self) -> bool:
+        """
+        If the model is a "direct model" - i.e., a model that directly estimates time of event from system state, rather than using state transition. This is useful for data-driven models that map from sensor data to time of event, and for physics-based models where state transition differential equations can be solved.
+
+        Returns:
```

Comment:
Does this docstring show up correctly in the API documentation? (Asking because the "Returns" is not "underlined")